static/js/app/cloudfoundry.js

```diff
@@ -0,0 +1,57 @@
+(function() {
+    // CloudFoundry Service
+    var cloudfoundry = function($http) {
+
+        // Get current authentication status from server
+        var getAuthStatus = function() {
+            return $http.get('/v2/authstatus')
+                .then(function(response) {
+                    return response.data.status;
+                }, function(response) {
+                    return response.data.status;
+                });
+        };
+
+        // Get organizations
+        var getOrgs = function() {
+            return $http.get('/v2/organizations')
+                .then(function(response) {
+                    return response.data.resources;
+                });
+        };
+        // Get organization spaces details
+        var getOrgSpaceDetails = function(org) {
+            return $http.get(org.entity.spaces_url)
+                .then(function(response) {
+                    var data = response.data;
+                    data.org_name = org.entity.name;
+                    return data;
+                });
+        };
+        // Get space details
+        var getSpaceDetails = function(space) {
```

Comment:
This should work the same way the `org` variable works. [List of spaces](https://github.com/18F/cf-console/blob/cfservice/static/js/app/partials/spaces.html#L2)